docs/gettinginvolved/infrastructure/documentation.rst

```diff
@@ -0,0 +1,112 @@
+Documentation Hosting
+=====================
+
+Flocker's documentation is hosted on S3, fronted by cloudfront and cloudflare.
```

Comment:
Someone would probably want me to say something about proper capitalization of product names.